phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java

```diff
@@ -28,7 +28,7 @@
     private SettingsManager settingsManager;
     private RepositoryManager repositoryManager;
 
-    VBox vBox = new VBox();
+    private VBox vBox = new VBox();
```

Comment:
If possible I would like it more if we do the initialization of `vBox` in the constructor